deps.ts

```diff
@@ -2,5 +2,5 @@ export {
   connectWebSocket,
   isWebSocketCloseEvent,
   WebSocket,
-} from "https://deno.land/std@0.57.0/ws/mod.ts";
-export * as DenoAsync from "https://deno.land/std@0.57.0/async/mod.ts";
+} from "https://deno.land/std@0.66.0/ws/mod.ts";
+export * as DenoAsync from "https://deno.land/std@0.66.0/async/mod.ts";
```

Comment:
```suggestion
} from "https://deno.land/std@0.81.0/ws/mod.ts";
export * as DenoAsync from "https://deno.land/std@0.81.0/async/mod.ts";
```
Sorry for not including this in my previous review; Standard lib update to the current version, 0.81.0, which should still be compatible with no breaking changes.